packages/flutter/test/cupertino/button_test.dart

```diff
@@ -464,6 +524,14 @@ void main() {
     expect(decoration.color, isSameColorAs(CupertinoColors.systemBlue.darkColor));
   });
 
+  testWidgets('All CupertinoButton const maps keys\' match the available style sizes', (WidgetTester tester) async {
+    for(final CupertinoButtonSize size in CupertinoButtonSize.values) {
```

Comment:
```suggestion
    for (final CupertinoButtonSize size in CupertinoButtonSize.values) {
```
Analyzer is complaining about this